gui/browser_osx.mm

```diff
@@ -177,6 +178,9 @@ - (IBAction) showHiddenFiles : (id) sender {
 		g_system->endGFXTransaction();
 	}
 
+	// Clear the event queue so that keyboard input (for example) isn't sent to ScummVM right after the browser closes
```

Comment:
This needs a much more extensive comment explaining this behaviour, since it isn't something expected.